### ISSUE TITLE
data producer

### ISSUE THREAD

**Issue by IshaqZOUAGHI** [gz101/kafka-spark-ELK-pipeline#10]
I've tried to run the project and the first issue I find is that the data producer (API) doesn't work.

**Comment by gz101** [comment]
Yep that's an issue with Heroku - they no longer have a free tier plan. I'm working on updating the documentation, but in the mean time here is the repo for the data producer. You can run it locally and make some tweaks to the pipeline code so that it works: https://github.com/gz101/geotech-parameters-api. Will update again once my documentation is updated.
